chained_swapper/models/chained_swapper.py

```diff
@@ -89,7 +89,9 @@ def _onchange_model_id(self):
     def write(self, vals):
         super().write(vals)
```

Comment:
```suggestion
        res = super().write(vals)
```
and return at the end

Reply:
Please do this also